src/main/java/com/fasterxml/jackson/databind/ser/std/StdJdkSerializers.java

```diff
@@ -48,12 +64,16 @@
         sers.put(Void.class, NullSerializer.instance);
         sers.put(Void.TYPE, NullSerializer.instance);
 
+        //And Sql Blob
+        sers.put(Blob.class, new SqlBlobSerializer());
+        sers.put(SerialBlob.class, SqlBlobSerializer.class);
```

Comment:
Not sure this is needed as it implements `Blob`? I guess unit test should show either way.
But if needed, should also be moved within try-block and register `SqlBlobSerializer.class` and not pre-create serializer.

Reply:
It is needed because when I use SerialBlob in the unit test, it will show error "No Serializer". I think this is because somehow it cannot comprehend that SerialBlob is actually implementation of Blob. If you have other suggestion, please do share.